src/zarr/core/metadata/v3.py

```diff
@@ -490,8 +492,11 @@ def to_numpy_shortname(self) -> str:
         }
         return data_type_to_numpy[self]
 
+    def to_numpy_dtype(self) -> np.dtype[Any]:
+        return np.dtype(self.to_numpy_shortname())
+
     @classmethod
-    def from_dtype(cls, dtype: np.dtype[Any]) -> DataType:
+    def from_numpy_dtype(cls, dtype: np.dtype[Any]) -> DataType:
```

Comment:
ditto https://github.com/zarr-developers/zarr-python/pull/2278/files#r1782030539

Reply:
modulo to/from of course